conf/logback.xml

```diff
@@ -17,7 +17,7 @@ https://www.playframework.com/documentation/2.4.x/SettingsLogger#Using-a-configu
         </rollingPolicy>
 
         <encoder>
-            <pattern>%date [%.-30thread] %logger[%file:%L] %highlight(%level: %msg%n%xException{3})</pattern>
+            <pattern>%date %logger[%file:%L] %level: %msg%n%xException</pattern>
```

Comment:
👍 

FYI, if you want colourful logging on your own machine, Mario shared this useful article with me: https://howchoo.com/g/mzm3ztdhm2u/how-to-colorize-your-logs-with-ccze